content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-assignments-in-github-classroom.md

```diff
@@ -0,0 +1,53 @@
+
+Optionally, you can enable feedback pull requests on anv assignment. If you enable this feature for an assignment, {% data variables.product.prodname_classroom %} creates a special pull request titled **Feedback** in the assignment repository for each student or group. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/leave-feedback-with-pull-requests)."
+
+If you integrate your {% data variables.product.prodname_classroom %} with an IDE, students can make changes to their assignment repositories through the IDE, or by checking out and running code locally with the appropriate software. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/integrate-github-classroom-with-an-ide)."
+
+You can reuse existing assignments in any other classroom you have admin access to, including classrooms in a different organization. For more information, see [AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/reuse-an-assignment)."
+
+You must create a classroom before you can create an assignment. For more information, see [AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/manage-classrooms#creating-a-classroom).
+
+## Types of assignments 
+
+You can either create a group assignment or n individual assignment in {% data variables.product.prodname_classroom %}.
+
+### Individual assignments
+
+An individual assignment is coursework for each student who participates in a course on GitHub Classroom. The student is intended to complete an individual assignment without help from other students.
+
+When a student accepts an assignment, GitHub Classroom automatically creates a new repository for the student. 
+
+For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-individual-assignment)."
+
+### Group assignments
+
+A group assignment is collaborative coursework for groups of students on GitHub Classroom. Students can work together on a group assignment in a shared repository, like a team of professional developers.
+
+When a student accepts a group assignment, the student can create a new team or join an existing team. GitHub Classroom saves the teams for an assignment as a set. You can name the set of teams for a specific assignment when you create the assignment, and you can reuse that set of teams for a later assignment.
+
+For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-a-group-assignment)."
+
+## Further reading
+
+- "[AUTOTITLE](/education/manage-coursework-with-github-classroom/create-an-individual-assignment)."
```

Comment:
```suggestion
- "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-individual-assignment)."
```

Updating the path to avoid redirects